secure_message/repository/saver.py

```diff
@@ -19,9 +20,9 @@ def save_message(domain_message, session=db.session):
         try:
             session.add(db_message)
             session.commit()
-        except Exception as e:
+        except SQLAlchemyError:
             session.rollback()
-            logger.error('Message save failed', error=e)
+            logger.exception('Secure message save failed')
```

Comment:
Any kind of id or context that can be added to help issue resolution?

Reply:
Its a valid point , I am not planning on changing it here , as I think there is some confusion as to what logging patterns we should be using . I am putting a separate ticket on our backlog to agree a format on our backlog .